sbe-tool/src/test/resources/since-deprecated-test-schema.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <sbe:messageSchema xmlns:sbe="http://fixprotocol.io/2016/sbe"
-                   package="since-deprecated"
+                   package="since.deprecated"
```

Comment:
This change breaks the C# build (see https://github.com/real-logic/simple-binary-encoding/runs/4171818266?check_suite_focus=true):
```
Error: SinceDeprecatedTests.cs(72,49): error CS0103: The name 'SinceDeprecated' does not exist in the current context [/home/runner/work/simple-binary-encoding/simple-binary-encoding/csharp/sbe-tests/sbe-tests.csproj]
Error: SinceDeprecatedTests.cs(73,49): error CS0103: The name 'SinceDeprecated' does not exist in the current context [/home/runner/work/simple-binary-encoding/simple-binary-encoding/csharp/sbe-tests/sbe-tests.csproj]
    0 Warning(s)
    42 Error(s)
```

Reply:
I see, sorry. Then I create a separate XML for this.